tmk_core/avr.mk

```diff
@@ -41,6 +41,11 @@ MCUFLAGS = -mmcu=$(MCU)
 EXTRALIBDIRS =
 
 
+# If your environment is WSL need this.Set arduino IDE for Windows10 path  include $$WINAVRDIR/bin/avrdude.exe and $$WINAVRDIR/etc/avrdude.conf
+WINAVRDIR="/mnt/c/tools/arduino/hardware/tools/avr/"
```

Comment:
Why are we doing this? 

AVRDUDE can be installed with `sudo apt install avrdude`, (in `/util/wsl_install.sh`), and that should be installed "globally" for WSL.   We should not be hard coding the path, as this will change EVERY time there is a PR (where somebody has set it to their config).